src/amo/components/AddonReviewCard/index.js

```diff
@@ -437,18 +436,22 @@ export class AddonReviewCardBase extends React.Component<InternalProps> {
               review={review}
               byLine={!this.isRatingOnly() && byLine}
```

Comment:
Interestingly, the byline is now hidden for rating-only reviews from the profile which means you can't get a permalink. Hmm. I think that's a bug. What do you think? We could fix it in a follow-up.

Reply:
Yes, seems like a bug. There are enough other changes required that I'll probably fix this too in this patch.

Reply:
The check for `!this.isRatingOnly()` is only needed when the `AddonReviewCard` is being displayed inside the `RatingManager`, so I removed this check and used css to hide the `byLine`. It works, but I'm not sure if that's the correct thing to do.